backup-tools/backup-restore/app.go

```diff
@@ -14,11 +14,9 @@ import (
 
 	"github.com/sapcc/containers/backup-tools/go-src/configuration"
 	"github.com/sapcc/containers/backup-tools/go-src/swiftcli"
-	"github.com/sapcc/containers/backup-tools/go-src/underscore"
 	"github.com/sapcc/containers/backup-tools/go-src/utils"
 
 	"github.com/ncw/swift"
-	"gopkg.in/urfave/cli.v1"
```

Comment:
Why removing the cli?

Reply:
I did not remove this, goimports removed this automatically when I saved the file. Therefore, the import must have been unnecessary.